### DISCUSSION TITLE
插件关闭时，higress gayeway日志报错，但插件实际还生效

### DISCUSSION THREAD

**Discussion by KingOfXi** [alibaba/higress#1512] [Q&A]
基于cluster-key-rate-limiter插件 进行二次开发，插件启用 关闭时会发现插件的限流规则还在生效，且higress-gateway是如下配置
![企业微信截图_17315844751490](https://github.com/user-attachments/assets/74335bd4-a50e-4936-9db0-795769140ed9)

cluster-key-rate-limiter 插件是如下 资源定义：
![企业微信截图_17315845131535](https://github.com/user-attachments/assets/93de8066-8c4c-4c98-b13d-c3d7e2d263e4)
![企业微信截图_17315845288635](https://github.com/user-attachments/assets/7ec5a79b-f4af-42b1-9ba8-aa1d82a4be61)
console配置如下图所示，一点击保存，按理说限流配置不应该生效
![企业微信截图_17315847133969](https://github.com/user-attachments/assets/bf95d940-4461-49f3-9909-e13ce83c4a09)

**Comment by KingOfXi** [comment]
不确定是插件的哪个配置导致这种问题，还是我基于插件二次开发的代码问题，若是后者，我则提供下改写的插件main.go类

**Reply by KingOfXi** [reply]
![企业微信截图_17316409872827](https://github.com/user-attachments/assets/5ca4ba8f-fa16-4416-9501-ed73b2b96374)

**Comment by CH3CHO** [comment]
![image](https://github.com/user-attachments/assets/bc6ab628-c6e7-4421-b8ff-b68eb7387e0e)

ai-token-ratelimit 这个你们打开了吗？如果打开了的话就关掉看一下。

**Reply by KingOfXi** [reply]
插件关闭时higress-gateway 会新增这些日志
![企业微信截图_17316458405679](https://github.com/user-attachments/assets/7d941f2d-e0ae-462d-8ab3-ef44e08d2de7)

**Reply by CH3CHO** [reply]
麻烦贴一下下面命令的执行结果：
```bash
kuebctl get wasmplugin ai-token-rate-limit-1.0.0 -n higress-system -o yaml
```

然后删掉这个资源看一下问题能否解决。
